internal/pkg/aws/ec2/ec2.go

```diff
@@ -83,8 +84,34 @@ func New(s *session.Session) *EC2 {
 	}
 }
 
-// ListVPC returns IDs of all VPCs.
-func (c *EC2) ListVPC() ([]string, error) {
+// VPC contains the ID and name of a VPC.
+type VPC struct {
+	ID   string
+	Name string
+}
+
+// String formats the elements of a VPC into a display-ready string.
+func (v *VPC) String() string {
+	if v.Name != "" {
+		return fmt.Sprintf("%s (%s)", v.ID, v.Name)
+	}
+	return v.ID
+}
+
+// ExtractVPC extracts the VPC ID from the VPC display string.
+func ExtractVPC(string string) VPC{
+	splitVPC := strings.Split(string, " ")
+	extractedVPC := VPC{
+		ID: splitVPC[0],
+	}
+	if strings.Contains(string, "(") {
+		extractedVPC.Name = strings.Trim(splitVPC[1], "()")
+	}
+	return extractedVPC
+}
+
+// ListVPCs returns names and IDs (or just IDs, if Name tag does not exist) of all VPCs.
+func (c *EC2) ListVPCs() ([]string, error) {
```

Comment:
If we always return the stringified VPC to the caller then they'll always be forced to call the `ExtractVPC` function. Instead, we can return a `VPC` struct and let callers decide if they want to call the `String` method on them.
```suggestion
func (c *EC2) ListVPCs() ([]VPC, error) {
```